app/src/main/res/values/strings.xml

```diff
@@ -40,4 +40,7 @@
 	<string name="blink_slow">Slow</string>
 	<string name="blink_normal">Normal</string>
 	<string name="blink_frenetic">Frenetic</string>
+
+	<string name="share">Share ...</string>
+	<string name="send_to">Send using</string>
```

Comment:
Where is is string used? I can't find it elsewhere in the project.
(Also, I'd be happy if the "Share ..." is changed to "Share…"   :)